website/react-magma-docs/src/pages/api/modal.mdx

```diff
@@ -212,6 +212,110 @@ export function Example() {
 }
 ```
 
+## Modal Header Ref
+
+The modal headerRef prop allows access to a header element within the Modal. This is helpful for when a custom header is needed as it provides the ability to focus on load.
```

Comment:
I think we can keep it generic to `Use the headerRef prop to get a reference to the header component`